tests/l2/README.md

```diff
@@ -113,10 +113,12 @@ $ oc logs intel-dgpu-hwinfo-l4572
 ```                        
 
 ### Verify Intel® QuickAssist Technology provisioning
-This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
+This workload runs [qatlib](https://github.com/intel/qatlib) sample tests using RPM packages. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
```

Comment:
using RedHat building and distributing QAT RPM package from repo codeready-builder-for-rhel-9-x86_64-rpms. (Correct me if the repo name is not accurate. )

Reply:
updated